pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -1,73 +1,43 @@
-{ stdenv, fetchurl, lib, qtbase, qtmultimedia, qtsvg, qtdeclarative, qttools, qtgraphicaleffects, qtquickcontrols2, full
-, libsecret, libGL, libpulseaudio, glib, wrapQtAppsHook, mkDerivation }:
+{ lib, buildGoModule, fetchFromGitHub, pkgconfig, libsecret }:
 
-let
-  version = "1.2.3-1";
-
-  description = ''
-    An application that runs on your computer in the background and seamlessly encrypts
-    and decrypts your mail as it enters and leaves your computer.
-
-    To work, gnome-keyring service must be enabled.
-  '';
-in mkDerivation {
+buildGoModule rec {
   pname = "protonmail-bridge";
-  inherit version;
+  version = "1.4.5";
 
-  src = fetchurl {
-    url = "https://protonmail.com/download/protonmail-bridge_${version}_amd64.deb";
-    sha256 = "032ggk9fvd19fbsqkzwzwh0hpyg8gpkrin71di7zsx6ias5innw1";
+  src = fetchFromGitHub {
+    owner = "ProtonMail";
+    repo = "proton-bridge";
+    rev = "br-${version}";
+    sha256 = "sha256-J7m1JcLrjUw0PIMoRWDXH1hax97/C7zq6Rj2o/SAaYw=";
   };
 
-  sourceRoot = ".";
+  vendorSha256 = "sha256-goaMQ4wmUoc/TLsIxmwH9C2VtKsnuwRmAs9C1sGosk0=";
 
-  unpackCmd = ''
-    ar p "$src" data.tar.xz | tar xJ
-  '';
+  nativeBuildInputs = [ pkgconfig ];
```

Comment:
```suggestion
  nativeBuildInputs = [ pkg-config ];
```

Reply:
In [all-packages.nix](https://raw.githubusercontent.com/NixOS/nixpkgs/master/pkgs/top-level/all-packages.nix) `pkgconfig` is an alias of `pkg-config` is there a reason to change ?

Reply:
It does not use the alias.

Reply:
Hmm. Are you sure about that ? I don't see why it does not use it. `protonmail-bridge` is called using `callPackage` inside `all-packages.nix` so it should have access to `pkgconfig` (which there is the same as `pkg-config`).

If there's something I'm not seeing here, can you explain it to me ?

Reply:
pkgconfig relies in the alias which may be removed in the future and you should avoid relying on it.

Reply:
It's fine to rely on an alias, we do that all the time. I didn't see any deprecation notice for `pkgconfig`. And even if in the future it is being removed we will get a treewide change so it's completely fine.

If you still believe it must be `pkg-config` instead of `pkgconfig`, I would like to receive a confirmation from a third-party (preference for a committer) or any precedent where the question was asked and the answer was decided.

Reply:
Ok, I asked on the unofficial Nix/NixOS Discord for more informations and they told me that `pkg-config` is the preferred way. I also see a lot of pull requests with requests to change `pkgconfig` to `pkg-config` so I'll assume that it is actually the preferred way.